files/nrpe/check_openstack.py

```diff
@@ -717,7 +711,7 @@ def check_floating_ips(self):
 
     SERVICE_TENANT_NAME="service"
     PUBLIC_NET_NAME="public"
-    public_network_id = self.neutron.list_networks(project_id=SERVICE_TENANT_NAME,
+    public_network_id = self.neutron.list_networks(project_name=SERVICE_TENANT_NAME,
```

Comment:
This looks like the thing that will make a certain capacity monitoring check work in pouta-devel. Yay!